src/gimelstudio/core/node/property.py

```diff
@@ -144,6 +145,29 @@ def CreateUI(self, parent, sizer):
         self.AddToFoldPanel(sizer, fold_panel, self.img)
 
 
+class ColorProp(Property):
+    """ 
+    Allows the user to select a color.
+    """
+    def __init__(self, idname, default=(255, 255, 255), visible=True):
+        Property.__init__(self, idname, default, fpb_label, expanded, visible)
+    
+    def CreateUI(self, parent, sizer):
+        fold_panel = self.CreateFoldPanel(sizer)
+        fold_panel.SetBackgroundColour(wx.Colour("#464646"))
+
+        self.colorPicker = ColorPickerButton(fold_panel,
```

Comment:
We use snake case variables, so it should be ``self.color_picker``. Technically, you don't even need to make it a class variable (with ``self``) as the widget object isn't needed anywhere else outside of the local scope.